tensorflow/python/ops/string_ops.py

```diff
@@ -123,29 +123,26 @@ def string_format(template, inputs, placeholder="{}", summarize=3, name=None):
   Example:
     Formatting a single-tensor template:
     ```python
-    tensor = tf.range(10)
-    formatted = tf.strings.format("tensor: {}, suffix", tensor)
-    expected = "tensor: [0 1 2 ... 7 8 9], suffix"
-
-    assert(formatted == expected)
+    >>> tensor = tf.range(10)
+    >>> formatted = tf.strings.format("tensor: {}, suffix", tensor)
+    >>> expected = "tensor: [0 1 2 ... 7 8 9], suffix"
+    >>> assert(formatted == expected)
```

Comment:
We don't need the assertion written explicitly. The following should work (modulo some output change):

```suggestion
    >>> formatted = tf.strings.format("tensor: {}, suffix", tensor)
    "tensor: [0 1 2 ... 7 8 9], suffix"
```

We should do the same for the other doctests below